src/load/azext_load/data_plane/utils/validators.py

```diff
@@ -333,3 +345,27 @@ def _validate_dimension_filter(string):
         comps = string.split("=", 1)
         result = {comps[0]: comps[1].split(",")} if len(comps) > 1 else {string: ""}
     return result
+
+
+def validate_split_csv(namespace):
+    if namespace.split_csv is None:
+        return
+    if not isinstance(namespace.split_csv, str):
+        raise InvalidArgumentValueError(
+            f"Invalid split-csv type: {type(namespace.split_csv)}"
+        )
+    if namespace.split_csv.casefold() not in [
+        "true",
+        "false",
+        "yes",
+        "no",
+        "y",
+        "n",
+    ]:
+        raise InvalidArgumentValueError(
+            f"Invalid split-csv value: {namespace.split_csv}. Allowed values: true, false, yes, no, y, n"
+        )
+    if namespace.split_csv.casefold() in ["true", "yes", "y"]:
+        namespace.split_csv = True
```

Comment:
any reason for not making this argument as a Boolean? Is it because its hard to discern from a "user provided `False`" with a "default `False`" ?